### ISSUE TITLE
Can't build amber app with new version

### ISSUE THREAD

**Issue by nilq** [amberframework/amber#1036]
### Description

I found this same issue, that should've been fixed with the newest release.

Trying to build my project with version 0.11.3, I get the following error:

```
08:32:48 Watch run  | (INFO) Building...
Error in src/amber-learning.cr:1: while requiring "../config/*"

require "../config/*"
^

in config/application.cr:1: while requiring "./database.cr"

require "./database.cr"
^

in config/database.cr:5: undefined method 'progname=' for Nil (compile-time type is (Logger | Nil))

Granite.settings.logger.progname = "Granite"
                        ^~~~~~~~

Rerun with --error-trace to show a complete error trace.
08:32:49 Watch run  | (INFO) Compile time errors detected, exiting...
```

### Steps to Reproduce

I'm trying to follow the Amber Auth example in the Gitbook.

After having run `amber new amber-learning -d sqlite && shards install`, `amber g auth User` and `amber db create migrate`, then trying to run `amber watch` I get the error.

**Expected behavior:** The project should build.

**Actual behavior:** The project doesn't build.

**Reproduces how often:** 100% of the time, after having tried two times.

### Versions

On Manjaro, with Crystal 0.27.0 and Amber 0.11.3:

```
☁  amber-learning [master] ⚡  amber -v
Amber CLI (amberframework.org) - v0.11.3
```

**Comment by jackturnbull** [comment]
See https://github.com/amberframework/amber/issues/1032 for more

**Comment by nilq** [comment]
@jackturnbull It was installed through AUR, but with a modified build script in order to install 0.11.3, the version is the same everywhere.

**Comment by jackturnbull** [comment]
Hmm, I'm a little lost in that case - I've just downloaded the 0.11.3 `.tar.gz` from the PKGBUILD and grep'd the source for a reference to `Granite.settings.logger.progname` and I've had no joy.

I've got no idea how, but the only thing that makes sense to me would be if you might have had some reference to the old 0.11.2 package version on your system at the point you generated the project - either that or your amber is reporting the wrong version.

All I can suggest in the short term is that you try and re-install amber from fresh (or use a binary compiled from master) and see if you're able to replicate?

**Comment by faustinoaq** [comment]
This issue was introduced on v0.11.2. I can't reproduce it on  v0.11.3

I guess this was fixed on latest version

> It was installed through AUR, but with a modified build script in order to install 0.11.3, the version is the same everywhere.

@nilq BTW, v0.11.3 is now available on AUR, sorry for late update :sweat_smile: 

https://aur.archlinux.org/packages/amber/

**Comment by jackturnbull** [comment]
I suspect this issue can be closed now.

**Comment by kvirani** [comment]
@nilq were you able to resolve the error on your machine? If so, how did you get around it?

**Comment by elorest** [comment]
This issue was introduced with 0.11.2 and was fixed in 0.11.3. Haven't ran into this issue in months.
